index.js

```diff
@@ -1,9 +1,13 @@
-var instructions = require('./instructions.json');
+module.exports = function(_language, _version) {
+    var version = _version || 'v5';
+    var language = _language || 'en';
 
-if (Object !== instructions.constructor) throw 'instructions must be object';
+    var instructions = require('./instructions').get(language);
+    if (Object !== instructions.constructor) throw 'instructions must be object';
 
-module.exports = function(_version) {
-    var version = _version || 'v5';
+    function capitalizeFirstLetter(string) {
+        return string.charAt(0).toUpperCase() + string.slice(1);
```

Comment:
This function isn't effective or desirable for all languages. Moreover, the overall instruction string may need to be placed in running text or title-cased depending on the context. Could capitalizing the first letter be a responsibility of client code rather than this library?